### ISSUE TITLE
PostgresSyntaxError: syntax error at or near "." on select_related

### ISSUE THREAD

**Issue by maitham** [collerek/ormar#455]
I get the following error `PostgresSyntaxError: syntax error at or near "."`

```
class Team(ormar.Model):
    class Meta:
        tablename: str = "team"
        database = database
        metadata = metadata

    id: UUID =  UUIDField(server_default=sqlalchemy.text("uuid_generate_v4()"), primary_key=True, index=True)
    name = ormar.Text(nullable=True)
    client_id = ormar.Text(nullable=True)
    client_secret = ormar.Text(nullable=True)
    created_on = ormar.DateTime(timezone=True, default =arrow.utcnow().datetime)

class User(ormar.Model):
    class Meta:
        tablename: str = "user"
        database = database
        metadata = metadata

    id: UUID =  UUIDField(server_default=sqlalchemy.text("uuid_generate_v4()"), primary_key=True, index=True)
    client_user_id = ormar.Text()
    token = ormar.Text(nullable=True)  
    team: UUID = ormar.ForeignKey(to=Team, name="team_id")

class Order(ormar.Model):
    class Meta:
        tablename: str = "order"
        database = database
        metadata = metadata

    id: UUID =  ormar.UUID(server_default=sqlalchemy.text("uuid_generate_v4()"), primary_key=True, index=True)
    user: Optional[Union[User, Dict]] = ormar.ForeignKey(User, name="user_id")

await User.objects.select_related("orders").all()
```

**Comment by collerek** [comment]
What is a `UUIDField`? You use it as primary key instead of `ormar.UUID` field.

You use the reserved sql keyword as table name (`order`) which is not recommended, but I think that should be handled - for Postgres all names should be quoted.

**Comment by collerek** [comment]
Should be fixed in 0.10.23 - please update and check
